src/pages/home/ReportScreen.js

```diff
@@ -309,9 +318,18 @@ class ReportScreen extends React.Component {
                             {/* Note: The report should be allowed to mount even if the initial report actions are not loaded. If we prevent rendering the report while they are loading then
                             we'll unnecessarily unmount the ReportActionsView which will clear the new marker lines initial state. */}
                             {(!this.isReportReadyForDisplay() || isLoadingInitialReportActions) && (
-                                <ReportActionsSkeletonView
-                                    containerHeight={this.state.skeletonViewContainerHeight}
-                                />
+                                <>
+                                    <ReportActionsSkeletonView
+                                        containerHeight={this.state.skeletonViewContainerHeight}
+                                    />
+                                    <ReportFooter
+                                        isComposerFullSize={false}
+                                        isOffline={this.props.network.isOffline}
+                                        onSubmitComment={() => {}}
+                                        report={{reportID: '0'}}
+                                        reportActions={{}}
```

Comment:
Similarly, these three (`onSubmitComment`, `report`, and `reportActions`) could just use a default when `shouldDisableCompose` is present - instead of having to add it in the two places.

Reply:
Looks great thanks.